bootstrap.sh

```diff
@@ -2,10 +2,12 @@
 
 apt-get update
 apt-get install python3 python3-pip -y
-pip3 install flask Flask-SQLAlchemy SQLAlchemy pymysql
+pip3 install -r /vagrant/requirments.txt
+
```

Comment:
Another PR removed this file. Please rebase and decide.
The pylint shows that the reason this command is failing is:
ERROR: Could not find a version that satisfies the requirement command-not-found==0.3 (from -r requirments.txt (line 17)) (from versions: none)
63
ERROR: No matching distribution found for command-not-found==0.3 (from -r requirments.txt (line 17))